src/index.ts

```diff
@@ -1,11 +1,11 @@
-import express, { Express, NextFunction, Request, Response, Router, response } from 'express';
-import dotenv from 'dotenv';
-import { ContactRepository } from './repositories/contactRepository';
-import { DbUtils } from '../utils/dbutils';
-const cors = require('cors');
+import express, { Express } from 'express';
+import cors from 'cors';
+// Allow async handlers to cleanly throw errors to the global error handler.
+import 'express-async-errors';
```

Comment:
Must import this package before routes. Allows us to handle errors in middleware, in a catch-all way, instead of per-route.

Reply:
OMG I'm so glad this exists. Gets me out of the business of hand-making errors.